meta/README.md

```diff
@@ -0,0 +1,5 @@
+# robocorp (metapackage)
+
+A package that includes the core Python libraries of Robocorp: [tasks](https://pypi.org/project/robocorp-tasks/), [logging](https://pypi.org/project/robocorp-log/), and [Control Room](https://robocorp.com/docs/control-room) integration.
```

Comment:
I think that it could also list the specific libraries here instead of just saying 'control room integration'.